ops/optimizer.rs

```diff
@@ -123,13 +123,8 @@ impl Transform {
           parse_quote! { *const #core::v8::fast_api::FastApiTypedArray<u32> };
 
         q!(Vars { var: &ident }, {
-          let var = match unsafe { &*var }.get_storage_if_aligned() {
-            Some(v) => v,
-            None => {
-              unsafe { &mut *fast_api_callback_options }.fallback = true;
-              return Default::default();
-            }
-          };
+          let var =
+            unsafe { (&*var).get_storage_if_aligned().unwrap_unchecked() };
```

Comment:
What kind of guarantee does Deno have that the fast api typed array will always be available? If not, it would panic.

Reply:
If you refer to the alignment, this here is sort of safe but not: V8 itself guarantees that all TypedArrays are always 32-bit aligned and based on some limited testing the real guarantee is actually that all TypedArrays will be 64-bit aligned on a 64-bit machine (tested on Linux only though).

An issue is, however, that Deno FFI makes it possible to make `ArrayBuffer`s with any alignment. So, while it's usually okay to trust that the alignment is correct it can theoretically be unsafe to do so. The unwrapping should be removed.

If you refer to the FastApiTypedArray struct actually being there? V8 guarantees that it always calls Fast API calls with existent TypedArray pointers and never with null pointers.